src/chains/ethereum/transaction/src/access-lists.ts

```diff
@@ -80,4 +81,77 @@ export class AccessLists {
       dataFeeEIP2930: dataFee
     };
   }
+
+  /**
+   * Creates a valid access list from a potentially invalid access list. Throws
+   * for invalid JSON-RPC data for any of the `address` values or entries of the
+   * `storageKeys` array.
+   * @param accessList
+   * @returns a valid access list that
+   *  1. Is an array
+   *  2. Each array element contains an object with the keys `address` and
+   *     `storageKeys`.
+   *  3. A value for the `address` key is not omitted.
+   *  4. The value for the `address` key is a 20-byte, hex-encoded string.
+   *  5. The `storageKeys` entry is an array.
+   *  6. Each entry of the `storageKeys` array is a 32-byte, hex-encoded string.
+   */
+  public static tryGetValidatedAccessList(accessList: AccessList): AccessList {
+    const validAccessList: AccessList = [];
+    if (!Array.isArray(accessList)) {
+      return validAccessList;
+    }
+    for (const accessListItem of accessList) {
+      const { address, storageKeys } = accessListItem;
+      if (!address) continue;
+
+      const validStorageKeys: string[] = [];
+      const validAccessListItem: AccessListItem = {
+        address: Data.from(address, ADDRESS_LENGTH).toString(),
```

Comment:
I was trying to figure out why we use Data.from instead of Address.from, and then discovered that the byte-length field isn't even used by `Data.from`: https://github.com/trufflesuite/ganache/blob/2affc8b3269532163b615b148bb437efda493418/src/packages/utils/src/things/json-rpc/json-rpc-data.ts#L9

@jeffsmale90, this may be of interest to you.

Reply:
I've added tests to cover this in #3784

Reply:
I'm glad `Data.from` works! But your initial question was valid, I often forget we have an `Address` class. I've switched over to use it rather than `Data`.